nemo_text_processing/inverse_text_normalization/en/taggers/cardinal.py

```diff
@@ -121,20 +123,24 @@ def __init__(self):
         )
 
         labels_exception = [num_to_word(x) for x in range(0, 13)]
-        graph_exception = pynini.union(*labels_exception)
+        labels_exception += [x.capitalize() for x in labels_exception]
+        graph_exception = pynini.union(*labels_exception).optimize()
 
         graph = (
             pynini.cdrewrite(pynutil.delete("and"), NEMO_SPACE, NEMO_SPACE, NEMO_SIGMA)
             @ (NEMO_ALPHA + NEMO_SIGMA)
             @ graph
-        )
+        ).optimize()
+
+        # accept semiotic spans that start with a capital letter
+        graph |= pynini.compose(TO_LOWER + NEMO_SIGMA, graph).optimize()
 
         self.graph_no_exception = graph
 
         self.graph = (pynini.project(graph, "input") - graph_exception.arcsort()) @ graph
 
         optional_minus_graph = pynini.closure(
-            pynutil.insert("negative: ") + pynini.cross("minus", "\"-\"") + NEMO_SPACE, 0, 1
+            pynutil.insert("negative: ") + pynini.cross(pynini.union("minus", "Minus"), "\"-\"") + NEMO_SPACE, 0, 1
```

Comment:
can we save this in variable instead of writing it twice? maybe check if we need consider capitalization?

Reply:
yes, liked `cased` for TN